lib/save_movie.dart

```diff
@@ -0,0 +1,29 @@
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+
+class Save extends StatefulWidget {
```

Comment:
I would change the name of this class to something that represents the widget itself not only the functionality of it

Reply:
changed to SaveMovieButton